Resources/views/CRUD/base_edit_form_macro.html.twig

```diff
@@ -1,35 +1,37 @@
 {% macro render_groups(admin, form, groups, has_tab) %}
     <div class="row">
 
-    {% for code in groups %}
-        {% set form_group = admin.formgroups[code] %}
+        {% for code in groups %}
```

Comment:
I think in twig you can do this directly : `{% for code in groups if admin.formgroups[code] is defined %}`